website/src/main/tut/docs/installation/cli.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+title: scalafix-cli
+---
+
+## scalafix-cli
+
+The recommended way to install the scalafix command-line interface is with
+[coursier](https://github.com/coursier/coursier#command-line).
+
+
+```sh
+// coursier
+coursier bootstrap ch.epfl.scala:scalafix-cli_@{{ site.scalaVersion }}:{{ site.stableVersion }} -f --main scalafix.cli.Cli -o scalafix
+./scalafix --help
+```
+
+```sh
+// homebrew
+brew install --HEAD olafurpg/scalafmt/scalafix
+scalafix --help
+```
+
+```sh
+// wget
+wget -O scalafix https://github.com/scalacenter/scalafix/blob/master/scalafix?raw=true
+./scalafix --help
+```
+
+Once the scalafix cli is installed, consult the --help page for further usage instructions.
+
+> TODO(gabro): automatically include help message?
```

Comment:
As a quick hack, we can add a tut code block that calls `println(helpMessage)` https://github.com/scalacenter/scalafix/blob/c0a26983514599172b30e515c9b2029cd92e1f64/scalafix-cli/src/main/scala/scalafix/cli/Cli.scala#L33

Reply:
This seems to do the trick

    ```tut:evaluated:plain
    println(s"\n${scalafix.cli.Cli.helpMessage}\n")
    ```

<img width="600" alt="screen shot 2017-10-09 at 14 45 05" src="https://user-images.githubusercontent.com/1408093/31338679-89e0e88e-ad00-11e7-9ca9-cee3b721ae31.png">

Reply:
nice! I think I can use a similar trick in other parts of the website.
I'll try to carve out some time tonight to address this and other easy fixes.